apps/crn-server/src/routes/manuscript.route.ts

```diff
@@ -91,12 +91,29 @@ export const manuscriptRouteFactory = (
 
     if (!loggedInUser || !userBelongsToTeam) throw Boom.forbidden();
 
-    const manuscript = await manuscriptController.create({
-      ...createRequest,
-      userId: loggedInUser.id,
-    });
+    try {
+      const manuscript = await manuscriptController.create({
+        ...createRequest,
+        userId: loggedInUser.id,
+      });
 
-    res.status(201).json(manuscript);
+      res.status(201).json(manuscript);
+    } catch (error) {
+      if (error instanceof Error) {
+        try {
+          const errorMessage = error.message as string;
+          const contentfulError = JSON.parse(errorMessage);
+          res.status(Number(contentfulError.status) || 500).json({
+            message: contentfulError.message || 'An error occurred',
+            errors: contentfulError.details?.errors || [],
+          });
+          return;
+        } catch {
```

Comment:
Hey, I think we need to add this to 

POST `/manuscripts/:manuscriptId`
PUT `/manuscripts/:manuscriptId`

because if you edit a manuscript and change the title to an existing title or submit a revised manuscript with an existing title it displays the default error, not the unique title one